introspection/org.freedesktop.NetworkManager.Device.OvsPort.xml

```diff
@@ -7,6 +7,16 @@
   -->
   <interface name="org.freedesktop.NetworkManager.Device.OvsPort">
 
+    <!--
+        Slaves:
+
+        Array of object paths representing interfaces which are currently slaved to
+        this port.
```

Comment:
The same. (I know, other device types use 'slaved' as well, but it sounds wrong to me).